### ISSUE TITLE
urlopen error [Errno 10054] The external host lost a connection

### ISSUE THREAD

**Issue by GillBill** [BitBotFactory/poloniexlendingbot#620]
After running Poloniex Lending Bot without problems for several months, last week it didn't work anymore. I get the following error <urlopen error [Errno 10054] The external host lost a connection> Requesting retourAvailableAccountBalances. There is nothing changed in the settings at Poloniex nor in the config file of the Lending Bot. 

See attachment for the complete error text

![error poloniexlendingbot](https://user-images.githubusercontent.com/37801984/37906256-fa206cf4-30f0-11e8-8fdd-b6ceb2a278fc.PNG)

Can anyone please explain what I need to do to solve this?

**Comment by rnevet** [comment]
Basically Poloniex is closing the connection, are you using IP filter? maybe your IP changed? otherwise is the problem persisting?

**Comment by Evanito** [comment]
I don't see a version number, which tells me you may be using an outdated version of the bot. 
Step 1 in troubleshooting our bot is to update to the most recent version. Please do so and get back to us! :)

If the problem doesn't come back, then it is no issue and may have just been temporary problems on Poloniex's end, such as if they were protecting against a DDoS attack.

**Comment by GillBill** [comment]
Thanx for your response. Unfortunatelly this is not the case, I even removed the restriction of the IP-address (temporarely) to see if this solves the problem, but the problem still persists.

Van: Raanan [mailto:notifications@github.com]
Verzonden: maandag 26 maart 2018 21:32
Aan: BitBotFactory/poloniexlendingbot <poloniexlendingbot@noreply.github.com>
CC: GillBill <gkorte@hotmail.com>; Author <author@noreply.github.com>
Onderwerp: Re: [BitBotFactory/poloniexlendingbot] urlopen error [Errno 10054] The external host lost a connection (#620)


Basically Poloniex is closing the connection, are you using IP filter? maybe your IP changed? otherwise is the problem persisting?

**Comment by Evanito** [comment]
I suspect the exchange may have in some way blacklisted your IP, hopefully temporarily. 
I cannot replicate this issue and it seems to be between you and the exchange.
You may want to generate a new API key and test if that works, or maybe just time will solve it.

If after nearly a week you are still experiencing this issue (without you doing anything that could have caused it) then the problem is probably not on your bot's end, but on your ISP blocking what they think is fraudulent traffic or Poloniex blocking what they think is DDoS.

If you find a solution on your own it would be great to hear it, but I do not think there is anything we can do on our end.
